src/main/java/core/basesyntax/service/operation/OperationStrategyImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private static final Map<FruitTransaction.OperationType, OperationHandler> map =
+            Map.of(FruitTransaction.OperationType.BALANCE, new BalanceSetOperationHandler(),
+                    FruitTransaction.OperationType.SUPPLY, new AdditionOperationHandler(),
+                    FruitTransaction.OperationType.RETURN, new AdditionOperationHandler(),
```

Comment:
Map with operation should be created in main and set here via constructor 
`supply` and `return` should be separate operations. 
Rename your operation according to their business logic (e.g.  `BalanceOperationHandler`, `ReturnOperationHandler` and etc.)